modules/grafana-agent/variables.tf

```diff
@@ -11,3 +11,8 @@ variable "external_labels" {
   type    = map(string)
   default = {}
 }
+variable "k8s_namespace" {
+  type        = string
+  default     = "default"
+  description = "Kubernetes namespace to use"
```

Comment:
This is only for the DaemonSet / Deployments - the service accounts cluster role{s, bindings} are not namespaced.

We should also add this deploys node-exporter in the same namespace - or ask for the namespace to deploy it in.